frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/resource-status.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import * as classNames from 'classnames';
 import { Badge } from '@patternfly/react-core';
+import * as classNames from 'classnames';
```

Comment:
@fhlavac was this change intentional?

Reply:
No, it wasn't . Missed it in the diff. Thank you :slightly_smiling_face:

Reply:
Yeah, the public folder is not properly linted... intentionally and by design (old code, lots of changes would be needed). So moving it into `packages` folder does prompt auto-fixing lint errors, especially those around import statements being ordered.